### PR TITLE
ncu-ci: add --cache and progress in display

### PR DESCRIPTION
**Pull request by joyeecheung**

### CODE REVIEW

**Comment by joyeecheung** [comment]
cc @nodejs/automation

File: lib/ci/ci_type_parser.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const CI_TYPES = new Map([
   [LITE_PR_PIPELINE, {
     name: 'Lite PR Pipeline',
     jobName: 'node-test-pull-request-lite-pipeline',
-    pattern: /job\/node-test-pull-request-lite-pipeline\/(\d+)\/pipeline/,
+    pattern: /node-test-pull-request-lite-pipeline\/(\d+)\/pipeline/,
```
Comment (richardlau) [inline lib/ci/ci_type_parser.js]: This is an unrelated change? (If it allows the GitHub bot posted links then that's 💯 👍 but should be in its own commit and probably have tests if that's the intention.)

**Comment by joyeecheung** [comment]
@richardlau Looks like I accidentally added some unrelated changes...anyway since I am at it, fixed https://github.com/nodejs/node-core-utils/issues/259 in the second commit
